library/include/hipblas.h

```diff
@@ -18159,6 +18283,32 @@ HIPBLAS_EXPORT hipblasStatus_t hipblasGemmStridedBatchedEx_v2(hipblasHandle_t
                                                               hipblasComputeType_t computeType,
                                                               hipblasGemmAlgo_t    algo);
 
+HIPBLAS_EXPORT hipblasStatus_t
+    hipblasGemmStridedBatchedExWithFlags_v2(hipblasHandle_t      handle,
+                                            hipblasOperation_t   transA,
+                                            hipblasOperation_t   transB,
+                                            int                  m,
+                                            int                  n,
+                                            int                  k,
+                                            const void*          alpha,
+                                            const void*          A,
+                                            hipDataType          aType,
+                                            int                  lda,
+                                            hipblasStride        strideA,
+                                            const void*          B,
+                                            hipDataType          bType,
+                                            int                  ldb,
+                                            hipblasStride        strideB,
+                                            const void*          beta,
+                                            void*                C,
+                                            hipDataType          cType,
+                                            int                  ldc,
+                                            hipblasStride        strideC,
+                                            int                  batchCount,
+                                            hipblasComputeType_t computeType,
+                                            hipblasGemmAlgo_t    algo,
+                                            hipblasGemmFlags_t   flags);
+
```

Comment:
Needs `#ifdef HIPBLAS_V2 #define hipblasGemmWithFlags hipblasGemmWithFlags_v2` around L20624 to keep with the pattern.